components/search/BookingInfoSearch/DatePickerWithRange.tsx

```diff
@@ -131,7 +141,20 @@ export function DatePickerWithRange({
             selected={dateRange}
             onSelect={handleDateRangeChange}
             numberOfMonths={2}
-            disabled={(date)=> isBefore(date, new Date() || isEqual(date, new Date()))}
+            disabled={(date) => {
+              const today = startOfDay(new Date());
+              const fromDay = dateRange?.from
+                ? startOfDay(dateRange.from)
+                : null;
+
+              // Disable dates before today and the same as 'from' if 'to' is not selected
+              return (
+                isBefore(startOfDay(date), today) ||
+                (fromDay !== null &&
+                  isEqual(startOfDay(date), fromDay) &&
```

Comment:
Yes, I believe so.

You can't select dates today or any date before.

Valid range: tomorrow (from today's date) to any future date (I believe up to 2 months since the calendar only goes that far).